docs/index.rst

```diff
@@ -79,9 +84,23 @@ pre-processing to time-series analytics & machine learning!
 License
 -------
 
-tempo is made available under ``databricks`` License. For more details, see
+Tempo is made available under ``databricks`` License. For more details, see
 `LICENSE <https://github.com/databrickslabs/tempo/blob/master/LICENSE>`_.
 
+Docs on Github Pages
+--------------------
+
+Tempo's `sphinx documentation is hosted on github pages <https://www.sphinx-doc.org/en/master/tutorial/deploying.html#id5>`_
```

Comment:
Should we link to the actual tempo docs instead of generic sphinx docs? Or perhaps both would be useful actually.

Reply:
@rportilla-databricks Do you want me to setup one separate page which talks about how we are deploying our docs with sphinx using github-pages and then have the link point to that page? 
The only reason I didn't have too much details is because I thought that it's beyond the scope of our work in tempo and if anyone wants to build there docs using sphinx they can follow the sphinx generic docs.

Reply:
No, this is fine.